libs/components/ag-grid/src/lib/modules/ag-grid/types/autocomplete-properties.ts

```diff
@@ -2,20 +2,62 @@ import { TemplateRef } from '@angular/core';
 import { SkyAutocompleteSelectionChange } from '@skyux/lookup';
 
 export interface SkyAgGridAutocompleteProperties {
+  /**
+   * Allows the user to specify arbitrary values not in the search results.
+   * @default false
+   */
   allowAnyValue?: boolean;
+  /**
+   * The static data source for the autocomplete cell to search when users enter text. For a dynamic data source such as an array that changes due to server calls, use search instead.
```

Comment:
```suggestion
   * The static data source for the autocomplete cell to search when users enter text. For a dynamic data source, such as an array that changes due to server calls, use search instead.
```
Also, the corresponding property description for `lookup-properties.ts` includes a sentence that says "You can specify static data, such as an array of objects, or you can pull data from a database." Should we include that here too?